### PR TITLE
Add spectator radios integration for EG and ACE spectators

### PR DESCRIPTION
**Pull request by mharis001**
**When merged this pull request will:**
- Add spectator radios UI for focused entities in the EG and ACE spectator displays
- Radios are added/removed by clicking entries in the list
- Add keybind to clear spectators radios list while in spectator displays
- Unfortunately a dialog to display all current spectator radios doesn't seem viable since we are unable to prevent mouse scroll/drag moving camera and mouse click selecting entities

<details>
<summary>Images</summary>

acre_spectator_1

acre_spectator_2

</details>

### CODE REVIEW

**Review by TheMagnetar — DISMISSED** [review]

**Comment by TheMagnetar** [comment]
Once the conflicts are resolved it can be merged in my opinion

**Review by jonpas — COMMENTED** [review]
I understand ACE Spectator will also automatically have that due to it extending EG Spectator?

File: addons/sys_spectator/stringtable.xml
Comment (jonpas) [inline addons/sys_spectator/stringtable.xml]:
```suggestion
            <English>No Radios</English>
```
Would look better I think.

**Comment by mharis001** [comment]
No ACE spectator will not, it is based on EG Spectator but does not extend it. Will add support for it in another PR; should be fairly straight forward but need to add a load event on ACE's side.

**Comment by jonpas** [comment]
Requires more research and figure out listening to radios that are currently transmitting (listening to outgoing transmissions).

**Comment by BrettMayson** [comment]
> Outgoing transmissions issue continues

This would be the similar to #685 ?

**Comment by jonpas** [comment]
It is the same cause I believe, but #685 is harder to solve, as we still care about signal processing, however we don't even want that for spectator.

**Comment by jokoho48** [comment]
> Outgoing transmissions issue continues

I played around with that for my spectator and we don't add own radios here
https://github.com/IDI-Systems/acre2/blob/f79d12e5180b11380efe5fff2a699619712e953a/addons/sys_core/fnc_processRadioSpeaker.sqf#L116
I think adding a simple `ACRE_IS_SPECTATOR` check here could fix it. Sadly I didn't found anyone to test that hypothesis

**Comment by jonpas** [comment]
A fix has already been conjured, but going to tag @Soldia1138 on above.

**Comment by Soldia1138** [comment]
That won’t work. But as @jonpas said, there’s already a fix in the pipeline

**Comment by jonpas** [comment]
Pushing back until ACE Spectator support exists.

**Comment by mharis001** [comment]
Added ACE Spectator support. Requires next ACE update which will add the loaded event but don't need to wait for that to merge (not a hard dependency).

**Review by TheMagnetar — APPROVED** [review]
LGTM

**Review by jonpas — APPROVED** [review]

**Pull request merged**
